src/components/DatasetItem.vue

```diff
@@ -261,17 +261,16 @@
    },
 
    apollo: {
-     opticalImageUrl: {
-       query: opticalImageQuery,
+     imageThumbnail: {
```

Comment:
This is still giving console warnings. It needs to be called `thumbnailImage`, not `imageThumbnail` to match the query name.